### ISSUE TITLE
Stop connection when bed heated

### ISSUE THREAD

**Issue by Hautdumont** [daid/LegacyCura#880]
Hello,
Everything is ok until bed and nozzle are heated at temp.
Then, everything stop to do anything and cannot continue.
I tried to stop computer many times, moved connection baudrate from 115200 to auto.
Nothing work.

I changed USB cable

HP Compaq 6710b
Seven 64 bit entreprise
2go ram
intel Dual core 2.00 Ghz
Printer on COM4

It worked before, I don't understand

**Comment by nophead** [comment]
Exactly what happens if the ground connection between the controller and its PSU is not good enough. When the bed current flows in it a small voltage develops on the controller's ground relative to mains ground. Some of the bed current then flows down the USB cable and corrupts the comms.

**Comment by Hautdumont** [comment]
Thanks, but there is no ground connexion, because there is a "two
connectors cable " only, like a shaver.
It seems to work a little better since I changed again baudrate from "auto"
to 115200.
But now, I don't trust in open source softwares because, the same problem
happened before with Repetier, that's why I changed to Cura.
I fear to print big parts now. Really , open source is not stable at all,
just cheap.
Thank you for your fast answer.
Best regards from France.
have a good weekend
Bruno

2014-06-28 16:44 GMT+02:00 Chris notifications@github.com:

**Comment by nophead** [comment]
If the USB connection drops when the bed turns it is hardware problem so I don't think you can blame open source software. Blame open source hardware instead!

What type of PSU are you using that doesn't need a ground connection? Most PSUs used for reprap (ATX or LED lighting PSUs), need one. Without it the casing becomes live due to the mains inlet filter capacitors so you get a shock if you touch it.

**Comment by nophead** [comment]
BTW, it is the 0V connection between the controller board and its PSU I was referring to, not the connection to mains earth. Having it floating would prevent any DC current down the USB cable but you would still have noise on it.

**Comment by Hautdumont** [comment]
Hello Chris,
This morning, I wanted to print landing skid for quadcopter: 3H50 to print
After 2 hours everything stop and head stay on part ---> If that is not a
crazy sensitive software , I don't know what it is.
I'm  Printer's technician for twenty years and I've never seen printers
doing that, same in USB.
So, my printer is ReprapPro Mendel .
I listened to you and just connected ground cable 1 hour ago . I check now
with 3 hours spare printing. >>> wait and see
Plus, I found Cura doesn't like to share with Repetier --> I use Repetier
to move the nozzle and check extrude material (I like Cura very much more
but I'm just little sad cannot control like Repetier)
When I connect with repetier, and disconnect, I have to unplug printer and
plug again because Cura cannot reoppen connexion. May be normal, I don't
know.
All what I can say is Cura is a very good program and very very faster to
slice and prepare printing.
Thank you for your very fast help
I will check next week how that is working with ground connexion and will
send email to you.
Best regards
Bruno

2014-06-29 14:21 GMT+02:00 Chris notifications@github.com:
